app/build.gradle

```diff
@@ -115,6 +115,11 @@ android {
     }
 
     sourceSets {
+        main {
```

Comment:
I would use same syntax as with other params:
```
main.res.srcDirs += ['src/main/res', 'src/main/res/font']
```

Reply:
Sure. I think this was inserted by some tool. I don't remember adding it.

Reply:
Looks as if this whole thing isn't even necessary. The font is picked up without this.